src/pages/Sketch/components/Canvas.js

```diff
@@ -283,7 +302,7 @@ function Canvas() {
         [point.x, point.y],
         [downPoint.x, point.y],
       ],
-      { strokeWidth: width, roughness: roughness }
+      fill === "true" ? { ...openShapeOptions, ...closedShapeOptions } : { ...openShapeOptions }
```

Comment:
Rather than repeating the same code at multiple places, it's better to extract it to a const and reuse it anywhere it is been used. Just like this:

```js
     const {{something}} = fill === "true" ? { ...openShapeOptions, ...closedShapeOptions } : { ...openShapeOptions }
     
     // Then use `something`, instead of re-writing the whole condition
```